install/pre-install-payload/payload.sh

```diff
@@ -83,13 +83,13 @@ function build_payload() {
 
 	docker manifest create "${extra_docker_manifest_flags}" \
 		"${registry}:${tag}" \
-		--amend "${registry}:x86_64-${tag}" \
-		--amend "${registry}:s390x-${tag}"
+		"${registry}:x86_64-${tag}" \
+		"${registry}:s390x-${tag}"
 
 	docker manifest create "${extra_docker_manifest_flags}" \
 		"${registry}:latest" \
-		--amend "${registry}:x86_64-${tag}" \
-		--amend "${registry}:s390x-${tag}"
```

Comment:
I'm curious why the `--amend` should be removed? Is it failing when there is nothing to amend, or what changed?

Reply:
I'm not 100% sure on why it's failing. It failed 3 weeks ago, as well. `docker manifest create` is an experimental feature, and the docs suggest (at least at this point) that the amend switch isn't needed.